CONTRIBUTING.rst

```diff
@@ -1,6 +1,11 @@
 Contribution guidelines
 =======================
 
+This guide is describing rules how to get your contributions to Anaconda. However, if you are seeking
```

Comment:
"This guide describes rules for how to get your contributions into Anaconda."

Reply:
Fixed.